lib/models/question.dart

```diff
@@ -28,28 +32,44 @@ class Question extends Equatable {
         displayType,
         imageUrl,
         coverImageOpacity,
-        coverImageUrl
+        coverImageUrl,
+        answers,
       ];
 
   factory Question.fromQuestionResponse(QuestionResponse response) {
     return Question(
       id: response.id,
       text: response.text ?? '',
       displayOrder: response.displayOrder ?? 0,
-      displayType: response.displayType ?? '',
+      displayType: enumFromString(DisplayType.values, response.displayType) ??
```

Comment:
Why do we use [JsonEnum](https://pub.dev/packages/json_serializable#enums)? 🤔

Reply:
@markgravity you're right, fixed by https://github.com/luongvo/flutter-survey/pull/65/commits/7040e97814651b590b0f1c7044342c5b4cb58ffb